WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -221,7 +221,11 @@ + (NSArray *)routes
 {
   FBElementCache *elementCache = request.session.elementCache;
   XCUIElement *element = [elementCache elementForUUID:request.parameters[@"uuid"]];
-
+  
+  //if element is not specified then use root element i.e. application
+  if(element == nil) {
```

Comment:
What is use case here? Endpoint requires element `:uuid`, so it does make sense to handle invalid uuid as valid use case.